### PR TITLE
Update driver-verifier-options.md

### PR DESCRIPTION
**Pull request by selerner**
@DOMARS can you take a look when you get a chance?

### CODE REVIEW

File: windows-driver-docs-pr/devtest/driver-verifier-options.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ Rule classes are marked with (\*) require I/O Verification (5) that will be auto
 | VM switch verification | 25 |
 | Code integrity checks | 26 |
 | Additional IRQL checking | 35 |
+| Driver Isolation checks (requires 36) | 33 |
```
Comment (tedhudek) [inline windows-driver-docs-pr/devtest/driver-verifier-options.md:66]: @selerner should the line with 33 be before the line with 35?

**Comment by selerner** [comment]
Whoops, yes

**Comment by DOMARS** [comment]
@selerner - Let me know if you would like me to make the number order fix. You might consider saying "Enable DIF".

**Comment by selerner** [comment]
That would be great if you could, thanks Don!

**Pull request closed without merging**